internal/metros/metro_test.go

```diff
@@ -0,0 +1,76 @@
+package metros
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCli_Metros(t *testing.T) {
```

Comment:
This branch needs to be rebased on the latest from `main` and this test has to be skipped in short mode.

Reply:
rebased